elixir/apps/api/lib/api/gateway/views/subject.ex

```diff
@@ -0,0 +1,10 @@
+defmodule API.Gateway.Views.Subject do
+  alias Domain.Auth
+
+  def render(%Auth.Subject{} = subject) do
+    %{
+      identity_name: get_in(subject, [Access.key(:actor), Access.key(:name)]),
+      actor_email: get_in(subject, [Access.key(:identity), Access.key(:email)])
```

Comment:
- `subject.actor.*` is always going to be there.
- Is `Access.key` required? I believe both `Map` and `Struct` implement the Access behavior.
- I think we're still missing the `identity_id` and `actor_id` here

Does the following work?

```suggestion
      identity_name: subject.actor.name,
      identity_id: get_in(subject, [:identity, :id]),
      actor_email: get_in(subject, [:identity, :email]),
      actor_id: subject.actor.id
```

Reply:
>     * Is `Access.key` required? I believe both `Map` and `Struct` implement the Access behavior.

Unit tests were failing otherwise.